markdown.dtx

```diff
@@ -1099,6 +1101,37 @@ end)()
 %  \begin{macrocode}
 local md5 = require("md5")
 %    \end{macrocode}
+% \begin{markdown}
+%
+% \pkg{KPathSea}
+%
+%:    A package that implements the loading of third-party Lua libraries
+%     and looking up files in the \TeX{} directory structure.
+%
+% \end{markdown}
+%  \begin{macrocode}
+local kpse
+(function()
+%    \end{macrocode}
+% \begin{markdown}
+%
+% If KPathSea has not been loaded before or if Lua\TeX{} has not yet
+% been initialized, configure KPathSea on top of loading it.
```

Comment:
```suggestion
% If \pkg{KPathSea} has not been loaded before or if Lua\TeX{} has not yet
% been initialized, configure \pkg{KPathSea} on top of loading it. Since
% \Hologo{ConTeXt} MkIV provides a `kpse` global that acts as a
% stub for \pkg{KPathSea} and the \pkg{lua-uni-case} library expects that
% `kpse` is a reference to the full \pkg{KPathSea} library, we load
% \pkg{KPathSea} to the `kpse` global.
```